### DISCUSSION TITLE
Full site editing: Parent and child themes

### DISCUSSION THREAD

**Discussion by paaljoachim** [WordPress/gutenberg#49667] [General]
I believe a discussion around the following topic would be helpful.

As I was doing the [FSE Program Exploration: Build a block theme](https://make.wordpress.org/test/2023/03/29/fse-program-exploration-build-a-block-theme/) I installed the Create Block Theme plugin.

Options which are seen are:
Export -active theme-
Create child of -active theme-
Clone -active theme-
Overwrite -active theme-
Create a blank theme
Create a style variation

I began thinking about why I have earlier created a child theme.
I did so because I needed to make larger changes and if I did so in the parent theme and it became updated my changes would be removed.

I then began to think about if I should clone the current theme be it Twenty Twenty Two or Twenty Twenty Three. If that would work alright in the long run and how that would affect the workings of the site.

I began two threads on Slack.

**[One in the FSE-outreach-experiment channel:](https://app.slack.com/client/T024MFP4J/C015GUFFC00/thread/C015GUFFC00-1680673808.384639)** 
Regarding yesterdays Hallways Hangout as well as the [FSE Program Exploration: Build a block theme](https://make.wordpress.org/test/2023/03/29/fse-program-exploration-build-a-block-theme/)
Using the [Create a Block plugin](https://wordpress.org/plugins/create-block-theme/):
Advantages/disadvantages to creating a child theme.
Use Twenty Twenty Two or Twenty Twenty Three and why?
Advantages/disadvantages to modifying a theme and then creating ones own theme.
I have my own ideas regarding the above but it would be nice with some thoughts from anyone reading this. I am thinking about creating a much shorter video to where I go through the Create a Block plugin.

**[I also asked a question in the core-themes channel.](https://wordpress.slack.com/archives/C02RP4VMP/p1680768148492039)** 
The question asked:
I am wondering about parent FSE themes, child themes and if parent themes need less updating compared to earlier themes. Should we still create a child theme in relation to Twenty Twenty Two / Twenty Twenty Three or might we as well just create a clone of a default theme as a new stand alone theme?

**Comment by carolinan** [comment]
What is the goal of this discussion?
To improve documentation about child themes?
Are there any actionable points?
What I mean is, if this is a documentation issue, should it not be in the documentation issue tracker?

**Comment by paaljoachim** [comment]
**The goal of this discussion.**
--> I am currently brain storming. Looking at parent and child themes from another perspective. I would like a discussion to where we look at the current approach og having a parent/child FSE themes, cloned theme or a blank theme. How can we make this easier? 

Let's say that we start out blank and are then able to import Twenty Twenty Two, Twenty Twenty Three or any other theme information into the blank theme.
Let's say that using one theme that there are ways to update it when a fix is released which would usually update one of the default themes. Meaning one can import the fix to add any features added to a theme.

**Documentation.**
--> In general looking at different perspectives/angles in how we can find ways to improve themes.

**Actionable points.**
--> Finding ways to make these things easier to handle.

Currently this started discussion is one of a few places where I am brain storming.

**Comment by carolinan** [comment]
Fixes are individual to themes, so I don't understand...
Something that is broken in my theme might not be broken in your theme, how would there be an overlap that could be imported?
Usually, the "fix" is to:

1. Open the single post in the block editor
2. Restore any broken blocks alternatively add a new block

_What are the specific things that break when a theme is updated?_

**Comment by paaljoachim** [comment]
I am trying to figure out if there is some common ground between a fix for a default theme and perhaps a cloned theme. 
Let's say that I create a clone of Twenty Twenty Three and make a very few adjustments. An update to Twenty Twenty Three is released and perhaps it would be something I also would need for the clone that I made of the theme. Perhaps not. 
Perhaps one really does not need a child theme and can just clone the newest default theme and use that.
